client/apps/edit/components/admin/article/index.coffee

```diff
@@ -13,6 +13,8 @@ module.exports = AdminArticle = React.createClass
     publish_date: moment().format('YYYY-MM-DD')
     publish_time: moment().format('HH:mm')
     focus_date: false
+    tier: @props.article.get('tier') || 2
+    featured: @props.article?.get('featured') || false
```

Comment:
i generally try to use coffeescript's `or` in places like this but not a blocker